api/app/tests/apps/signals/test_filters_v1.py

```diff
@@ -0,0 +1,297 @@
+from datetime import datetime, timedelta
+
+from django.db import connection
+from rest_framework.test import APITestCase
+
+from signals.apps.signals.models import CategoryAssignment, Signal, Status
+from signals.apps.signals.workflow import BEHANDELING, ON_HOLD
+from tests.apps.signals.factories import (
+    LocationFactory,
+    PriorityFactory,
+    ReporterFactory,
+    SignalFactory,
+    StatusFactory,
+    SubCategoryFactory
+)
+from tests.apps.users.factories import SuperUserFactory
+
+
+class TestFilters(APITestCase):
+    """ Tests filters on the private v1 list endpoint: /signals/v1/private/signals """
+    SIGNALS_DISTRIBUTE_DAYS_CNT = 10
+    SIGNALS_DISTRIBUTE_HOURS_CNT = 10
+    SUBCATEGORY_CNT = 5
+    LIST_ENDPOINT = '/signals/v1/private/signals/'
+
+    signals = []
+    sub_categories = []
+
+    @classmethod
+    def _create_signals(cls):
+        cls.signals = [SignalFactory.create() for _ in
+                       range(cls.SIGNALS_DISTRIBUTE_DAYS_CNT + cls.SIGNALS_DISTRIBUTE_HOURS_CNT)]
+
+    @classmethod
+    def _set_signal_created_updated_at(cls, signal_id: int, dt: datetime):
```

Comment:
We already use freezegun in our tests. This way you can freeze the date/time for specific tests. So for example you can use it in the following way when creating signals:

```
from freezegun import freeze_time
with freeze_time("2018-01-14"):
    signal = SignalFactory()
```

Reply:
Ah nice one, will implement this!